src/faq/troubleshooting.rst

```diff
@@ -0,0 +1,222 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _faq/troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+Where are the CouchDB log files located?
+----------------------------------------
+
+For a default Linux/Unix installation the log files are located here::
+
+    /usr/local/var/log/couchdb/couch.log
+
+This is set in the default.ini file located here::
+
+    /etc/couchdb/default.ini
+
+If you've installed from source and are running CouchDB in dev mode the log
+files are located here::
+
+    YOUR-COUCHDB-SOURCE-DIRECTORY/tmp/log/couch.log
+
+How do I use transactions with CouchDB?
```

Comment:
I like the spirit of this example but it has some bad practices in it for 2.x, including using JS views (Mango is faster and easier to set up), emission of the document ID and rev (which are always in the output of a view anyway, use of the first person (who is talking?), and doesn't explain why a reduce is necessary or what changes when you add a reduce to a JS view - amongst other smaller complaints. It also says that it's an off-the-cuff response, which feels inappropriate within the context of official documentation.

Personally I'd drop it as it stands. It's a common RDBMS question, though, and it makes me want to have somethign about this in the docs at some point.